packages/InteractiveProfilingTool-Tests.package/IPTTreeModelTestCase.class/instance/testSourceCodeWhenClassDoesNotExistAnymore.st

```diff
@@ -0,0 +1,23 @@
+as yet unclassified
+testSourceCodeWhenClassDoesNotExistAnymore
+
+	| messageTallyMock compiledMethod tree source |
```

Comment:
I saw you use `tallyMock` in an earlier test. Please be consistent in the variable naming.